tests/hazmat/primitives/test_aead.py

```diff
@@ -257,9 +254,9 @@ def test_nonce_too_long(self, backend):
     def test_params_not_bytes(self, nonce, data, associated_data, tag_length,
                               backend):
         key = AESCCM.generate_key(128)
-        aesccm = AESCCM(key)
         with pytest.raises(TypeError):
-            aesccm.encrypt(nonce, data, associated_data, tag_length)
+            aesccm = AESCCM(key, tag_length)
```

Comment:
This is a parametrized test so it raises in both now. But really it should be split into a separate test so let me do that.